.github/workflows/model-registry-upstream-sync.yml

```
@@ -0,0 +1,524 @@
+          fi
+
+      - name: Commit conflict state for PR
+        if: steps.outcome.outputs.status == 'conflict'
+        run: |
+          mkdir -p sync-artifacts
+          git add -A
+          if ! git diff --cached --quiet; then
+            git commit -m "WIP: model-registry upstream sync conflict (resolve before merge)
+
+Automated sync stopped with conflicts. Resolve markers, then continue with:
+npm run update-subtree -w packages/model-registry -- --continue"
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)